### ISSUE TITLE
Move files from `~/.comicbox` to standard locations

### ISSUE THREAD

**Issue by buckley-w-david** [comicbox/comicbox#53]
The [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) specifies environment variables and defaults for where to store user specific application data.

The relevant parts for us would be:

**$XDG_DATA_HOME**
> $XDG_DATA_HOME defines the base directory relative to which user specific data files should be stored. If $XDG_DATA_HOME is either not set or empty, a default equal to $HOME/.local/share should be used.

For the sqlite database and probably the certificate.

**$XDG_CONFIG_HOME**
> $XDG_CONFIG_HOME defines the base directory relative to which user specific configuration files should be stored. If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used.

For the configuration file.

**Comment by zwzn** [comment]
I didn't know those existed, we should change to use them

**Comment by buckley-w-david** [comment]
Yeah, I didn't know either, just found out from a blog post complaining about developers not complying with the standard and littering the authors home directory with dotfiles
